.gitignore

```diff
@@ -21,3 +21,4 @@ docs/api
 build-target
 flink-staging/flink-avro/src/test/java/org/apache/flink/api/io/avro/generated/
 atlassian-ide-plugin.xml
+/notes/*
```

Comment:
Can you remove this entry? Seems very specific to your setup, not a generic entry.